core/tests/Test.Microsoft.Identity.Core.Unit/Mocks/MockHelpers.cs

```diff
@@ -60,6 +60,11 @@ public static string CreateClientInfo()
 
         public static string CreateClientInfo(string uid, string utid)
         {
+            if (String.IsNullOrEmpty(uid) && String.IsNullOrEmpty(utid))
```

Comment:
>if (String.IsNullOrEmpty(uid) && String.IsNullOrEmpty(utid)) [](start = 12, length = 60)

does clientinfo exists with out both uid and utid? at least i would expect uid should always be there... but actually I would expect both. thus should this be an || instead of &&?

Reply:
Yes, I would expect both as well. We don't really construct it like this through the code, only via deserialization from json. Theoretically you can have one of those 2 members be empty / null.